src/fielddef.ts

```diff
@@ -250,6 +252,11 @@ export function defaultType(fieldDef: FieldDef, channel: Channel): Type {
 export function normalize(fieldDef: ChannelDef, channel: Channel) {
   // If a fieldDef contains a field, we need type.
   if (isFieldDef(fieldDef)) { // TODO: or datum
+    // TODO: swojit - you can normalize bin here
+    if (!!fieldDef.bin) {
+      fieldDef.bin = isBoolean(fieldDef.bin) ? {maxbins: 10} : fieldDef.bin;
```

Comment:
(Actually, there is one case I forgot to change it to prevent side effect, please fix that too.)